package.json

```diff
@@ -24,19 +24,43 @@
     "url": "https://github.com/silvermine/videojs-chromecast/issues"
   },
   "homepage": "https://github.com/silvermine/videojs-chromecast#readme",
-  "devDependencies": {
+  "browserify": {
+    "transform": [
+      "browserify-shim"
+    ]
+  },
+  "browserify-shim": {
+    "video.js": "global:videojs"
+  },
+  "dependencies": {
+    "browserify-shim": "3.8.14",
```

Comment:
Should this be a `devDependency`?

https://github.com/thlorenz/browserify-shim#you-will-always